deployment/README.md

```diff
@@ -213,6 +284,8 @@ should be sufficient.
 
 Fluentd is deployed with no replicas. Once you have ElasticSearch
 running as desired, scale fluentd to every node to feed logs into ES.
+The example below would be for a cluster with 3 nodes; substitute the
+actual number of nodes in your cluster.
 
     oc scale dc/logging-fluentd --replicas=3
```

Comment:
Not in this change, but why do we provide examples of both 'rc/loggin' and 'dc/logging'